resources/tag_models/page.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Yep, agree, I'm going to make these the same format/structure, it's going to be easier for the enrichment processes to have things in the same place too.